ert/ensemble_evaluator/builder/_ensemble.py

```diff
@@ -18,6 +19,7 @@
 from ._realization import _Realization
 
 if TYPE_CHECKING:
+    import asyncio
```

Comment:
is import of asyncio really related to type checking?

Reply:
If the only usage is

`"asyncio.Queue[CloudEvent]"`

then it will appear as such, I think…